forms-flow-web/src/components/Modals/ImportModal.js

```diff
@@ -114,19 +114,10 @@ const ImportModal = React.memo(({ importFormModal, onClose, formSubmitted,
                             {importError && <span className="upload-status-error">{importError}</span>}
                         </div>
                         {importError && importError.includes("already exists") &&
-                            <div className="import-error-note">
-                                <div className="d-flex gap-2 align-items-center">
-                                    <IButton />
-                                    <Translation>{(t) => t("Note")}</Translation>
-                                </div>
-                                <div>
-                                    <Translation>
-                                        {(t) => t(`If you want to replace an existing form,
+                        <CustomInfo Heading="Note" Content='If you want to replace an existing form,
```

Comment:
pls cross check this . casing of 'heading' and 'content' is wrong

Reply:
Updated.